enhancements/kube-apiserver/audit-policy.md

```diff
@@ -287,6 +288,27 @@ The profile translate like this:
         # catch-all rule to log all other requests with request and response payloads
         - level: RequestResponse
     ```
+ - `UserRequests`
+
+     ```yaml
+         ... # common head
+
+       # Don't log for oauth tokens
+       - level: None
+         resources:
+         - group: "oauth.openshift.io"
+         resources: ["oauthaccesstokens"]
```

Comment:
Can we split the oauthaccesstoken change for all policies (as discussed last week) from the filtering by `system:authenticated:oauth`.

Reply:
Splitting is fine. Since this change is not related to the `userrequest policy` in isolation and effects all prior and existing policies to be updated, I'm happy to do this in a separate PR post-userrequest add if that is desired.